app.py

```diff
@@ -13,7 +14,7 @@
 
 @app.route('/')
```

Comment:
again, why do we need this route?

Reply:
so we can render our index.html file (homepage), as well as create our DB tables for the first time (if do not exist already)

Reply:
Remember what we discussed? do you want, each time someone visits the main page ('/'), to invoke this method?